.golangci.yaml

```diff
@@ -171,7 +171,6 @@ linters:
     - wastedassign
     - whitespace
     - wsl
-    #  - exhaustivestruct
```

Comment:
We could remove this commit from the backport, but it may make future backports easier.